src/Data/Poly.hs

```diff
@@ -32,9 +32,15 @@ module Data.Poly
   , pattern PolyOverFractional
   , unPolyOverFractional
 #endif
+  -- * Laurent interface
```

Comment:
Let us keep the interface of `Data.Poly` as is; it is enough to expose `Laurent` from their own module `Data.Poly.Laurent`.